modules/payloads/singles/bsd/x64/shell_reverse_tcp.rb

```diff
@@ -9,7 +9,7 @@
 
 module Metasploit3
 
-  CachedSize = 108
+  CachedSize = :dynamic
```

Comment:
Determining if a payload is :dynamic is a little precarious. It currently just generates the payload a few times and sees if the size changes, but this is not always reliable. We probably need a way to force a manual override.